src/Commands/MakeFactoryCommand.php

```diff
@@ -29,20 +29,20 @@ public function handle(): void
     {
         $this->info(string: 'Generating factory & interface file to your project');
 
-        $this->resolveGenerateForFactoryInterface();
+        $this->resolveFactoryContract();
 
-        $this->resolveGenerateForFactoryConcrete();
+        $this->resolveFactoryConcrete();
     }
 
     /**
      * @throws FileNotFoundException
      */
-    protected function resolveGenerateForFactoryInterface(): void
+    protected function resolveFactoryContract(): void
     {
-        $fileName = 'Abstract' . $this->resolveNameArgument() . '.php';
+        $fileName = $this->resolveNameArgument() . 'Contract' . '.php';
```

Comment:
since the Contract and .php is same why use concatenate?

Reply:
lol i didn't notice it